app/screen/EnterRoomScreen.tsx

```diff
@@ -1,12 +1,117 @@
-import React from 'react';
-import {View, Text} from 'react-native';
+import React, { useState } from 'react';
+import { View, Text, Pressable, StyleSheet, SafeAreaView } from 'react-native';
+import EnterRoomButton from '../component/enter-room/EnterRoomButton';
+import EnterRoomTextForm from '../component/enter-room/EnterRoomTextForm';
+import EnterRoomBackground from '../component/enter-room/EnterRoomBackground';
+import global, { COLORS } from '../styles/global';
+import GuessWhomLogo from '../component/GuessWhomLogo';
 
 export default () => {
+    const [shownForm, setShownForm] = useState<"Join" | "Create" | "">("");
```

Comment:
you might want to look into typescript interface

Reply:
Added enum